src/modules/flow/iio/nodes.c

```diff
@@ -275,8 +275,10 @@ gyroscope_open(struct sol_flow_node *node, void *data, const struct sol_flow_nod
 
     mdata->iio_base.config.buffer_size = opts->buffer_size;
     mdata->iio_base.config.sampling_frequency = opts->sampling_frequency;
-    snprintf(mdata->iio_base.config.sampling_frequency_name,
+    ret = snprintf(mdata->iio_base.config.sampling_frequency_name,
```

Comment:
@yongli3 yes, you are right. my mistake. 

So if you write 29 bytes. but ret return less than 29 bytes will not go to err.